_data/people/efajardo.yml

```
@@ -71,3 +71,32 @@ presentations:
   meetingurl: https://indico.fnal.gov/event/24246/
   focus-area: doma
   project: caching
+- title: Stashcache: CDN for Science
```

```suggestion
- title: "Stashcache: CDN for Science"
```

Space followed by colon is not allowed in YAML.